src/main/resources/reference.conf

```diff
@@ -511,6 +511,41 @@ resourceTypes = {
     }
     reuseIds = true
   }
+  spend-profile = {
+    actionPatternObjects = {
+      "create" = {
+        description = "Can create profiles"
+      }
+      "update" = {
```

Comment:
what is being updated? It is a best practice to be granular in defining actions.

Reply:
So you would recommend making an update permission for every updatable field in the data structure?
That would be name, billingAccountId, and description.

I guess i just don't know where the edge of granularity is. At this point we only have one update operation.

Reply:
Updated to two update actions: update_billing_account and update_metadata.